docs/introduction/about.md

```diff
@@ -5,66 +5,58 @@ sidebar_label: What is Calimero?
+- **Cross-shard transactions**: Calimero enables [cross-shard](https://docs.calimero.network/bridge/bridging/cross_shard_calls)  transactions, empowering you to interact with third parties on the public chain or other permissioned enterprise shards.
 
-- Public smart contracts if setup can also call into private shard’s contracts (extended trust model, similar to Oracles).
+- **Interacting with public blockchain applications**: Leveraging [Calimero SDKs](https://docs.calimero.network/calimero_sdk/installation), smart contracts deployed on private shards can interact with public blockchains (extended trust model, similar to Oracles), expanding the reach and capabilities of your application.
 
-- An encrypted channel can be used for private shards to interact with each other. 
+- **Encrypted channel for private shard interaction**: Calimero supports secure and encrypted channels for private shards to interact with each other, safeguarding sensitive data and ensuring privacy.
 
 
 ## Benefits of using private shards by Calimero network
 
 Calimero’s private shards have all the benefits of NEAR in terms of high performance, security and sustainability. But as an important distinction, shards are not using NEAR’s validators, execution and state storage, but rather you’re using your own. The main benefits include:
 
-1. **Scalability**: Scaling your business by having access to free transactions within the shard.
+1. **Scalability**: Calimero's private shards offer unlimited scalability, enabling businesses to handle increased transaction volumes and accommodate future growth. Enjoy the benefits of free transactions within your shard, supporting your business expansion.
 
-2. **Security**: Making part (or all) of your transactions private, with only people/entities with the access to the shard being able to view the data.
+2. **Security**: With private shards, you can make certain transactions or data accessible only to authorized parties, ensuring confidentiality and privacy. This controlled access allows you to protect sensitive information and maintain data integrity.
 
-3. **EVM compatibility**: You can use DApps already built on Ethereum mainnet instead of building them from scratch.
+3. **EVM compatibility**: Private shards on Calimero are compatible with Ethereum Virtual Machine (EVM), enabling seamless integration with existing Ethereum-based decentralized applications (DApps). This compatibility saves time and resources by leveraging the existing Ethereum ecosystem.
 
-4. **WASM compatibility & Javascript support**:  This makes is for easy onboarding to web3
+4. **WASM compatibility & Javascript support**: Calimero supports WebAssembly (WASM) compatibility and JavaScript, making it easy for developers to onboard and leverage web3 technologies. You can quickly build and deploy applications on Calimero with familiar programming languages.
 
```

Comment:
Calimero supports WebAssembly (WASM) and JavaScript, providing developers with a seamless onboarding experience. It allows you to use familiar programming languages to develop and deploy applications on Calimero, enabling you to start leveraging web3 technologies.